duecredit/collector.py

```diff
@@ -195,20 +197,65 @@ def cite(self, entry, **kwargs):
 
         return citation
 
+    @staticmethod
+    def _args_match_conditions(conditions, *fargs, **fkwargs):
+        """Helper to identify when to trigger citation given parameters to the function call
+        """
+        hit_default = False
+        for (arg, kwarg), values in iteritems(conditions):
```

Comment:
sure, but `arg` here is not the argument, rather it's the position of the argument, right?